_posts/2018-09-28-3D-Printing-Intro.md

```diff
@@ -0,0 +1,400 @@
+When using SLM instead, the metal particles are actually fully molten and they fuse together much more firmly creating extremely though objects!
+
+Objects created with these methods are free of internal stresses and defects that are common and hard to prevent with other production methods.
+The parts can also be printed as a whole rather than needing assembly of different parts, which further reduces the possibility for failures or errors during assembly.
+This however comes at great cost, literally, since these methods are very expensive and require state of the art equipment!
+Because of this high cost they are used in industries where the fault tolerance is very low, such as aerospace.
+
+
+### Different printing materials
+
+As you might have noticed in the previous section different printing methods require different printing materials.
+In the section below we'll go into more detail about these materials.
+
+#### Polymers/plastics
+
+FDM and DLP printers require polymer based materials.
+These are plastic or plastic like materials that are easy to print.
+
+*PLA (PolyLactic Acid):*
+
+- Biologically degradable
+- Melting (printing) temperature: 170 ⇔ 230 °C
+- Can be used to print medical supplies
+- Many colors available + glow in the dark
+- Not UV stable
+- Cannot withstand high temperatures, do not leave it in the sun inside a car!
+
+*ABS (Acrylonitrile Butadiene Styrene):*
+
+- Not biologically degradable
+- Melting (printing) temperature: 220 ⇔ 260 °C
+- Many colors available + translucent
+- UV semi stable
+- Can withstand higher temperatures than PLA!
+- Harder to print than PLA and requires venting since the fumes are toxic!
+
+*PETG (PolyEthylene Terephthalate Glycol):*
+
+- Not biologically degradable
+- Combines best of PLA and ABS
+- Melting (printing) temperature: 220 ⇔ 250 °C
+- Many colors available + translucent
+- UV semi stable
+- Can withstand higher temperatures than PLA!
+- No toxic fumes, odorless
+
+These three materials are commonly used in FDM printers. 
+The most and safest to use is PLA, which is fully bio degradable (over a long long time), it also is the easiest material to print with.
+ABS is stronger but not biodegradable and produces toxic fumes when printing, it also requires very precise cooling during printing or the print will warp and the layer will detach, ruining the print.
+PETG is the best of both worlds, it is stronger than PLA and does not produce toxic fumes.
+There are a lot more materials available, oftentimes with very specific properties to make it act more like rubber, be more flexible, glow in the dark,...
+All these different materials require different print settings and some can be quite hard to get right, experimentation is key!
+Some materials also exist that can be used in a medical context, these are however not printed with the average FDM printer since they need to match very high standards for medical use (both internal and external body usage).
```

Comment:
I am not sure about the (both internal and external body usage phrase ... 

maybe this => for both internal and external use